docs/technical_reference/property_models/modified_ADM1.rst

```diff
@@ -199,6 +201,8 @@ Kinetic Parameters
    ":lime:`Magnesium coefficient for polyphosphates, Mg_PP`", ":math:`Mg_{PP}`", "Mg_PP", 1/3, ":math:`\text{dimensionless}`"
    "Carbon dioxide acid-base equilibrium constant, K_a_co2", ":math:`K_{a,co2}`", "K_a_co2", 4.94e-7, ":math:`\text{kmol/}\text{m}^3`"
    "Inorganic nitrogen acid-base equilibrium constant, K_a_IN", ":math:`K_{a,IN}`", "K_a_IN", 1.11e-9, ":math:`\text{kmol/}\text{m}^3`"
+   "Molar concentration of hydrogen, S_H", ":math:`S_{H}`", "S_H", 3.4e-8, ":math:`\text{kmol/}\text{m}^3`"
+   "Molar concentration of hydroxide, S_OH", ":math:`S_{OH}`", "S_OH", 3.4e-8, ":math:`\text{kmol/}\text{m}^3`"
```

Comment:
I am not sure these should be regarded as parameters since these would essentially tie to pH or pOH, which would technically vary as a function of composition.

Reply:
@adam-a-a So maybe I just delete these two lines and call it a day since the "Value at 20C" doesn't really mean much, and these terms are defined along with their equations under "Additional Constraints". We could add a new section called "Additional Variables" but it wouldn't really add any additional information imo.

Reply:
I agree that it wouldn't add information, but to me it's more a matter of categorizing correctly.